src/firefly/js/ui/DropDownMenu.jsx

```diff
@@ -25,7 +25,7 @@ function placeDropDown(e,x,y, beforeVisible) {
 export function SingleColumnMenu({children}) {
     return (
         <DropDownMenu>
-            <Stack direction='column'>
+            <Stack direction='column' sx={{'.ff-toolbar-button':{width:'100%', justifyContent:'flex-start'}}}>
```

Comment:
`width:'100%'` can be replaced with `width:1`.

Reply:
interesting. Yes I remember that now.